### PR TITLE
Fix turning from alt screen to normal screen and vice versa.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Fix turning from alt screen to normal screen and vice versa. 
Fix https://github.com/sourcelair/xterm.js/issues/229.
Target branch: buffer-set

### CODE REVIEW

**Comment by AndrienkoAleksandr** [comment]
@parisk can you look?

File: src/InputHandler.ts [hunk 1]
```diff
@@ -1440,8 +1441,8 @@ export class InputHandler implements IInputHandler {
    *   Save cursor (ANSI.SYS).
    */
   public saveCursor(params: number[]): void {
-    this._terminal.savedX = this._terminal.buffer.x;
-    this._terminal.savedY = this._terminal.buffer.y;
+    this._terminal.buffers.active.x = this._terminal.buffer.x;
```
Comment (parisk) [inline src/InputHandler.ts]: `this._terminal.buffers.active` is the same with `this._terminal.buffer`.

I am not sure yet though if the `savedX` and `savedY` cursor positions have to be stored in the buffer or universally in the terminal.

Any thoughts?

**Comment by parisk** [comment]
Thanks @AndrienkoAleksandr! This seems to be working.

Can you take a look at https://github.com/sourcelair/xterm.js/pull/728/files#r123939766 though, as I will need to change this piece of code?

File: src/InputHandler.ts (hunk 1 above)
Comment (AndrienkoAleksandr) [inline src/InputHandler.ts]: Ok I will look.

**Pull request merged**